eggo/fabric_util.py

```diff
@@ -99,6 +99,12 @@ def _install_adam():
             run('mvn clean package -DskipTests')
 
 
+def _install_adam_partitioning():
+    run('mkdir -p /root/adam-partitioning')
+    with cd('/root/adam-partitioning'):
+        run('wget https://github.com/tomwhite/adam-partitioning/raw/master/lib/adam-partitioning-0.0.1-SNAPSHOT-job.jar')
```

Comment:
Is this a temporary thing until the patches merge in ADAM/kite?  If this is to be permanent, I'd support putting this in either ADAM or here.